CHANGELOG.md

```diff
@@ -146,6 +146,11 @@
     * Shopper Insights (BETA)
         * Remove `BraintreeClient` from constructor
         * Update package name to `com.braintreepayments.api.shopperinsights`
+    
+## v4 unreleased
+
+* PayPal
+    * Fix issue that causes a JSON parsing error when Pay Later is selected during checkout.
```

Comment:
I think this release went out this afternoon - can we grab the full CHANGELOG from main?

Reply:
good catch! i just pushed the update.